src/test/java/uk/gov/hmcts/reform/sscs/transformers/SscsCaseTransformerTest.java

```diff
@@ -208,10 +233,34 @@ public void givenBenefitTypePipWithIsOtherBenefit_thenErrorMessage() {
         pairs.put(BenefitTypeIndicatorSscs1U.UC.getIndicatorString(), false);
         pairs.put(BENEFIT_TYPE_OTHER, "Attendance Allowance");
         CaseResponse result = transformer.transformExceptionRecord(sscs1UExceptionRecord, false);
-        assertFalse(result.getErrors().isEmpty());
+        assertTrue(result.getErrors().size() == 1);
         assertEquals("is_benefit_type_pip and benefit_type_other have contradicting values", result.getErrors().get(0));
     }
 
+    @Test
+    public void givenBenefitTypePipWithIsOtherBenefitYes_thenErrorMessage() {
+        pairs.remove("is_benefit_type_pip");
+        pairs.put(BenefitTypeIndicatorSscs1U.PIP.getIndicatorString(), "No");
+        pairs.put(IS_BENEFIT_TYPE_OTHER, "Yes");
+        pairs.put(BenefitTypeIndicatorSscs1U.ESA.getIndicatorString(), "Yes");
+        pairs.put(BenefitTypeIndicatorSscs1U.UC.getIndicatorString(), "Yes");
+        CaseResponse result = transformer.transformExceptionRecord(sscs1UExceptionRecord, false);
+        assertTrue(result.getErrors().size() == 1);
+        assertEquals("is_benefit_type_esa, is_benefit_type_uc and is_benefit_type_other have contradicting values", result.getErrors().get(0));
+    }
+
+    @Test
+    public void givenBenefitTypePipWithIsOtherBenefitTrue_thenErrorMessage() {
```

Comment:
Not sure this test name makes sense. Mentions PIP but the test has nothing to do with PIP - can you update?